docs/javascript/v2/release-notes/release-notes.mdx

```diff
@@ -14,6 +14,30 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hls-player                  | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhls-player.svg)](https://badge.fury.io/js/%40100mslive%2Fhls-player)           |
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+# 2023-12-01
+Released: `@100mslive/hms-video-store@0.10.23`, `@100mslive/react-sdk@0.8.23`, `@100mslive/hls-player@0.1.23`, `@100mslive/roomkit-react@0.1.14`
+
+### Fixed:
+- Prebuilt: Audio only config is removed to prevent accidental selection which results in black tiles in the call
+
+### Added:
+- Prebuilt now supports more chat based controls like pausing, hiding messages, blocking peers, and private/public messaging which can be configured from the dashboard
+- Name input in the preview is now disabled if the `userName` prop is provided to the prebuilt component
+
+# 2023-11-28
+Released: `@100mslive/hms-video-store@0.10.22`, `@100mslive/react-sdk@0.8.22`, `@100mslive/hls-player@0.1.22`, `@100mslive/roomkit-react@0.1.13`
+
+### Fixed:
+- Resolved "Leave called before join error" which occurred intermittently during preview
+- Fixed audio stats showing up in the video section of subscriber stats
```

Comment:
did we mention subscriber stats earlier?

Reply:
No, should I remove this?